<launch>
	<arg name="rviz" default="true" />

	<rosparam file="$(find map_manager)/cfg/occupancy_map_param.yaml" ns="/occupancy_map"  />
	<rosparam file="$(find map_manager)/cfg/object_map_param.yaml" ns="/occupancy_map"  />
	<param name="use_sim_time" value="true" />
	<node pkg="map_manager" type="occupancy_map_node" name="occupancy_map_node" output="screen" />

	<node name="save_lidarMap_node" pkg="map_manager" type="save_lidarMap_node" output="screen">
        <param name="save_lidarMap" value="false"/>
    </node>

    <group if="$(arg rviz)">
      <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find map_manager)/rviz/sim_map.rviz" />
    </group>
</launch>